src/apps/authentication/components/AuthenticationRequired.tsx

```diff
@@ -46,6 +46,13 @@ const mapStateToProps = (state: any) => ({
   profile: profileSelector(state),
   profileLoading: getUserDetailsLoadingSelector(state)
 });
+const location = {
+  pathname: '/login'
```

Comment:
@jwamalwa  did you miss a coma on this line we would have some thing like  pathname: '/login',